source/modules/MEI.shared.xml

```diff
@@ -1317,6 +1317,17 @@
       <memberOf key="att.pitched"/>
     </classes>
   </classSpec>
+  <classSpec ident="att.keyMode" module="MEI.shared" type="atts">
```

Comment:
Thinking about the proposed changes for `keySig`/`key` it might be better to name it `att.key.anl`.

Reply:
We currently don't have any other `att.key.*` attributes, but the other `att.key*` classes we have (`att.keyAccid`, `att.keySig`, att.keySigDefault`) all have the domain appended. So perhaps `att.keyMode.anl`?

Reply:
To be resolved in context of #1194 in MEI 6